lib/src/room.dart

```diff
@@ -242,8 +256,7 @@ class KurentoRoom extends ChangeNotifier implements Room {
 
     await _messageSubscription?.dispose();
 
-    await _mq.unsubscribe(_serviceRequestPresenceTopic);
-    await _mq.unsubscribe(_participantTopic);
+    _mq.dispose();
```

Comment:
We don't have to unsubscribe from the individual topics, since `dispose` disconnects from the broker and deletes all subscriptions.